Source/RageCommunity.Library.TestPlugin/Commands.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Rage;
+using Rage.Attributes;
+using RageCommunity.Library.Wrappers;
+
+namespace RageCommunity.Library.TestPlugin
+{
+    public static class Commands
+    {
+        [ConsoleCommand("Test the native wrappers for vehicle horns.")]
+        public static void Command_RageLibraryHornTest()
+        {
+            if (Game.LocalPlayer.Character.CurrentVehicle != null)
```

Comment:
Just an FYI if you didn't know, you can call `!Game.LocalPlayer.Character.CurrentVehicle` to do a null check.